17thblog/blog.php

```diff
@@ -0,0 +1,17 @@
+<html>
+	<head>
+		<title> This is the blog part</title>
+		<link rel='stylesheet' href='style.css'/>
+	</head>
+	<body class = 'bg'>
```

Comment:
as a general style thing, I wouldn't put spaces between attribute names and values. This isn't really code in the way that PHP is, and while I think most browsers understand what you have here, I don't think it's technically "correct"

Reply:
Good to know. So instead of:

<body class = 'bg'>

it should be <body class='bg'>  ?

Reply:
I think some formatting got stripped out from your comment.

Reply:
It did. Not sure why it did that. My bad. Instead of "<body class =  'bg'> it should be <body class='bg'> ?

Reply:
It did, my bad. For some reason when I type anything with the less than or greater than symbol it doesn't show up. I meant to say this. 

So instead of: body class = 'bg'  
it should be body class='bg'   ?

And in the title, there shouldn't be a space in front of "This"?

Reply:
It's probably interpreting those as html tags and stripping it out. Try putting three \` signs before and after code to format it like this: ```<this is a test>```

Reply:
but as far as what you were asking, yes, `class='bg'` is correct

Reply:
Great, thanks! Will update my code.